### PR TITLE
Feat: xcm router weights

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description
Allows to set the weights of the xcm router more granulary. This is especially importatnt to get a realistic max weight for the pov.

### CODE REVIEW

**Review by NunoAlexandre — DISMISSED** [review]
LGTM! Overwriting the router on dev is fine with me 👍

**Review by wischli — DISMISSED** [review]
AFAICT, a migration for upgrading DEV is not required as we can apply the changes of `XcmDomain` to the router by calling `LiquidityPoolsGateway::set_domain_router`. 

If we want to update existing storage without manually overwriting, we would need a migration of course as the newly added `max_pov_size` will default to `0` otherwise.

**Comment by mustermeiszer** [comment]
@cdamian I did adapt our approach. I think we need to use specific fees and weights everywhere instead of setting defaults. LMKWYT.

File: pallets/liquidity-pools-gateway/routers/src/lib.rs [hunk 1]
```diff
@@ -197,19 +210,7 @@ where
 	/// Sets the weight information for the provided XCM domain location, and
 	/// the fee per second for the provided fee asset location.
 	pub fn do_init(&self) -> DispatchResult {
-		pallet_xcm_transactor::Pallet::<T>::set_transact_info(
```
Comment (cdamian) [inline pallets/liquidity-pools-gateway/routers/src/lib.rs:200]: Did we test to confirm that none of these are needed or do we manually need to set these storages after creating a router?

Comment (mustermeiszer) [reply pallets/liquidity-pools-gateway/routers/src/lib.rs:200]: Non are needed. The `pallet-xcm-transactor` can work with user provided values and only fills in the `Option` weights and fees if they are not provided with the defaults.

Comment (mustermeiszer) [reply pallets/liquidity-pools-gateway/routers/src/lib.rs:200]: I tested that I still sends with `Okay`

Comment (mustermeiszer) [reply pallets/liquidity-pools-gateway/routers/src/lib.rs:200]: There is a test for that I mean

**Comment by cdamian** [comment]
I will pick this up and fix the integration tests, will update this branch when ready.

**Comment by mustermeiszer** [comment]
I can not approve. this one.. But looks good.

**Comment by mustermeiszer** [comment]
I guess you can approve @cdamian

**Comment by cdamian** [comment]
@mustermeiszer I'll re-base, approve and enable auto-merge.

**Review by cdamian — APPROVED** [review]

**Pull request merged**